core/shared/src/main/scala/org/threeten/bp/Duration.scala

```diff
@@ -61,16 +61,16 @@ object Duration {
   lazy val ZERO: Duration = new Duration(0, 0)
 
   /** Constant for nanos per second. */
-  private def NANOS_PER_SECOND: Int = 1000000000
+  private final val NANOS_PER_SECOND = 1000000000
```

Comment:
Are we sure these don't affect js size?

Reply:
It's hard to reason about this.

Previously, every time this constant was used it would require a function call to `NANOS_PER_SECOND`. This requires a function `NANOS_PER_SECOND` to be emitted in the JS. This also means it's harder for the JS runtime to optimize it.

Now the generated JS code will never have `NANOS_PER_SECOND` in it. This is because before emitting JS code the Scala compiler will replace all the uses directly with `1000000000`.